tools/codegen/gentabledefs.py

```diff
@@ -89,6 +89,15 @@ def __repr__(self):
     "kernel_required": "KERNEL_REQUIRED",
 }
 
+# When generating C++ structs for a table defintion, avoid field that conflict with C / C++ / ObjectiveC keywords
+CPP_KEYWORDS = [ "protected", "class", "auto", "int", "virtual", "double", "float", "try", "catch",
+    "cast", "case", "const", "default", "delete", "continue", "concept", "char", "break",
+    "do", "export", "explicit", "public", "private", "synchronized", "volatile", "while", "signed",
+    "template", "inline", "interface", "namespace","not","friend","extern", "mutable"
+]
+
+MISLABELED_SPECS={"wifi_scan":"wifi_survey"
```

Comment:
Maybe instead of having this, let's just fix the labeling. I am also responsible for this as I landed that table. Sorry.